src/components/TextField/TextField.tsx

```diff
@@ -410,6 +410,12 @@ export function TextField({
     'aria-multiline': normalizeAriaMultiline(multiline),
   });
 
+  const textFieldStyles = classNames(
+    styles.Backdrop,
+    newDesignLanguage && connectedLeft && styles.ConnectedLeft,
+    newDesignLanguage && connectedRight && styles.ConnectedRight,
+  );
```

Comment:
Then this would become:
```
const backdropClassName = classNames(
    styles.Backdrop,
    newDesignLanguage && connectedLeft && styles['Backdrop-connectedLeft'],
    newDesignLanguage && connectedRight && styles['Backdrop-connectedRight'],
  );
```